docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -46,28 +46,39 @@ Running an Origin
 StashCache origin containers may be run with either multiple mounted host partitions (recommended) or a single host
 partition.
 
+It is recommended to use a container orchestration service such as [docker-compose](https://docs.docker.com/compose/)
+or [kubernetes](https://kubernetes.io/) whose details are beyond the scope of this document.
+The following sections provide examples for starting origin containers from the command-line as well as a more
+production-appropriate method using systemd.
+
+
 !!!important "Partition ownership"
     Host partitions mounted into the StashCache origin container must be owned `10940:10940`
 
 ### Multiple host partitions (recommended) ###
 
-For improved performance and storage, we recommend multiple partitions for handling namespaces (HDD), data (HDDs),
-and metadata (SSDs).
+For improved performance and storage,
+especially if your origin is serving over 10 TB of data,
+we recommend multiple partitions for handling namespaces (HDD, SSD, or NVME), data (HDDs), and metadata (SSDs or NVME).
 
 ```console
 user@host $ docker run --rm --publish 1094:1094 \
              --publish 1095:1095 \
-             --volume <HDD NAMESPACE PARTITION>:/xcache/namespace \
-             --volume <SSD METADATA PARTITION 1>:/xcache/meta1
+             --volume <NAMESPACE PARTITION>:/xcache/namespace \
```

Comment:
Is this true that the inside partition for origin is called `/xcache` I do not know if this would lead to someone getting confused.

Reply:
Eh, if we just tell them where to place things I'm not worried about it. However, I'm not sure if the namespace/meta/data locations make sense for an origin. If a site has data in a given partition, should they just mount that partition to `/xcache/namespace/`?

Reply:
Not really for an origin we should just have a namespace.

Reply:
If we ask them to mount their data into `/xcache/namespace`:

1. How do we support multiple partitions? It's fine if the answer is "we shouldn't"
1. Is it harmful if we have a `oss.space meta` and `oss.space data` directives in these cases or will XRootD just not use them?

Reply:
ahh yea I think if you want multiple partions you need meta and data. I just have never ran into that scenario that someone would like to run an origin with JBOD. Cause since it is not raided data loss is a huge possibility

Reply:
I think it's fine if we abandon the idea. I think I should be able to update the container appropriately.

Reply:
For an origin the multiple disks I think makes no sense.

Reply:
https://github.com/opensciencegrid/docker-xcache/pull/47

Reply:
The latest `fresh` origin containers have been updated to only support the single disk mode and I've updated the origin doc appropriately.